src/owners_file.rs

```diff
@@ -43,6 +44,7 @@ impl OwnersFileConfig {
         text: &str,
         path: P0,
         repo_base: P1,
+        seen_owners: &mut HashMap<PathBuf, Option<PathBuf>>,
```

Comment:
Lets expand the name slightly, to improve readability. My first thought when seeingthe variable was that it would include the groups/users, rather than paths, until I looked more closely at the type.
```suggestion
        seen_owners_files: &mut HashMap<PathBuf, Option<PathBuf>>,
```